mlir/include/mlir/Dialect/Rock/Tuning/RockTuning.h

```diff
@@ -35,8 +35,20 @@ struct TunableParams {
 };
 
 TunableParams *createTunableParamSpace(ModuleOp &mod);
-
 bool tuningSetParam(ModuleOp &mod, ParamEntry *paramEntry);
+bool tuningSetStr(ModuleOp &mod, std::string perfConfig);
+
+struct TuningTable {
+  std::map<std::string, std::pair<std::string, float>> tuningMap;
```

Comment:
`llvm::StringMap` perhaps, since we have it?

Reply:
So, I tried to see how to use it and found this post
https://stackoverflow.com/questions/49031588/what-to-prefer-a-llvmstringmap-or-stdmap-during-code-generation-using-llvm
An answer points the stringmap is slower than std::map. Not sure it's still relevant but we can stick with std::map for this implementation.